js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -288,12 +288,14 @@ export class MiniAppBridge {
    * This function returns access token details from the host app.
    * (provided the rakuten.miniapp.user.ACCESS_TOKEN is allowed by the user)
    * It returns error info if user had denied the custom permission
+   * @param {string} audienceRequested the audience the MiniApp requests for the token
+   * @param {string[]} scopesForAudience the associated scopes with the requested audience
    */
-  getAccessToken() {
+  getAccessToken(audienceRequested?: string, scopesForAudience?: string[]) {
     return new Promise<AccessTokenData>((resolve, reject) => {
       return this.executor.exec(
         'getAccessToken',
-        null,
+        { audience: audienceRequested, scopes: scopesForAudience },
```

Comment:
Why use the property names `audienceRequested` and `scopesForAudience`? If you just use `audience` and `scopes`, you could simplify passing the data here because you can use shorthand property names:
```suggestion
        { audience, scopes },
```

Reply:
done